packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/wdio-reporters.3/SpecReporter.tool.mdx

```diff
@@ -1,3 +1,47 @@
 # Spec Reporter
 
-The Spec Reporter writes test results to file in JSON format and is enabled by default in the `terra-functional-testing` wdio configuration.
+The spec reporter is a custom reporter plugin that aggregates test data and writes the results to file. The generated results file will contain all relevant metadata for the test run in JSON format.
+
+Example Output:
+
+```json
```

Comment:
out of scope for this update, but is is it possible to capture and write this output to the docs, that way the code + example are in sync?

Reply:
It's probably possible, but I think dynamic docs might be a little too tricky for the value it provides.  I had a hard time coming up with useful information to add to this document so I thought a sample output might provide some value, I'm still a little on the fence about it.